src/ui.tsx

```diff
@@ -156,6 +158,9 @@ class UserInterface extends Component<Props, State> {
       thumbnailHeight: thumbnailSizes[2].size,
       selectMultipleImagesMode: false,
     };
+
+    /* eslint-disable @typescript-eslint/unbound-method, @typescript-eslint/no-unsafe-assignment */
+    this.addUploadedImage = this.addUploadedImage.bind(this);
```

Comment:
Is there a way to make it work on methods without doing this? ie. keep the methods as arrow functions?

Reply:
Yes, I'd like to get rid of that, but I cannot find a way to get decorators to work with arrow functions.